packages/core/src/generators/imports.ts

```diff
@@ -234,6 +234,11 @@ export const addDependency = ({
 
       if (types) {
         if (values) {
+          for (let i = types.length - 1; i >= 0; i--) {
```

Comment:
instead of using splice would you be ok to recreate and array without the duplicated type? If we have a side effect from this it could be difficult to debug otherwise.